azurerm/data_source_arm_scheduled_time.go

```diff
@@ -0,0 +1,172 @@
+package azurerm
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+type Frequency string
+
+const (
+	Day     Frequency = "Day"
+	Hour    Frequency = "Hour"
+	Month   Frequency = "Month"
+	OneTime Frequency = "OneTime"
+	Week    Frequency = "Week"
+)
+
+func dataSourceArmScheduledTime() *schema.Resource {
```

Comment:
could we add some documentation for this data source? the relevant files are `./website/d/scheduled_time.html.markdown` and `./website/azurerm.erb`

Reply:
Done!